.circleci/config.yml

```diff
@@ -109,8 +109,10 @@ jobs:
     environment:
       RUSTFLAGS: -D warnings
     steps:
+      - checkout
       - run: git submodule sync
       - run: git submodule update --init --depth=1
+      - run: rustup component add clippy
```

Comment:
We should add clippy to the docker image instead.
Or maybe, do the whole clippy job in GitHub CI because it's easier to set up there.

Reply:
I have updated the docker image to include clippy, so this step is no longer needed.